AI/StupidAI/StupidAI.cpp

```diff
@@ -117,7 +122,10 @@ BattleAction CStupidAI::activeStack( const CStack * stack )
 	if(stack->type->idNumber == CreatureID::CATAPULT)
 	{
 		BattleAction attack;
-		static const std::vector<int> wallHexes = {50, 183, 182, 130, 78, 29, 12, 95};
+		static const std::vector<int> wallHexes =
+		{
+			50, 183, 182, 130, 78, 29, 12, 95

```

Comment:
So here we have bit longer "list initializer"... When looking at it I do not find it to be particularly easier to read, and if the benefit is saving space then it will not help for really long value lists or when they are not short numbers.